docs/src/network_analysis/crn_theory.md

```diff
@@ -0,0 +1,332 @@
+> Note, currently API functions for network analysis and conservation law analysis
+> do not work with constant species (currently only generated by SBMLToolkit).
+
+# [Chemical Reaction Network Theory](@id network_analysis_structural_aspects)
+
+The systems of ODEs or stochastic chemical kinetics models that arise from chemical
+reaction networks can often have their steady-state properties (number of steady states, etc.) known in advance,
+simply by analyzing the graph structure of the network. The subfield of chemistry
+and math studying this relationship is called [Chemical Reaction Network Theory](https://en.wikipedia.org/wiki/Chemical_reaction_network_theory).
+
+In this tutorial we give a broad overview of chemical reaction network theory, building on the discussion of the structure of 
+the reaction network ODEs in the previous section. We also introduce several of the Catalyst API functions for network
+analysis. A complete summary of the exported functions is given in the API
+section
+[Network Analysis and Representations](@ref api_network_analysis).
+
+Broadly, results from chemical reaction network theory relate a purely
+graph-structural property (e.g. deficiency) to dynamical properties of the reaction system
+(e.g. complex balance). The relevant graph here is the one corresponding to the [reaction complex representation](@ref network_analysis_reaction_complexes)
```

Comment:
Here I'd make the `deficiency` and `complex balance` links to corresponding sections (if nothing else to mark that these are terms we do not necessarily expect the reader to know at this stage).